docs/metadata.md

```diff
@@ -579,12 +579,69 @@ Any Golang repository processed by Cachito will contain both modules and package
 [go-package type][go-package-type]). The Content Manifest lists only the packages. That
 is usually good enough. Modules for dependencies *are* listed in [sources](#sources).
 
+## SBOM
+
+The SBOM (full name Software bill of materials) is a document that describes the
+content present in a container image. A JSON Schema specification is available in the
+CycloneDX repository): [sbom.json].
+
+Cachito itself knows nothing about container images. The document returned from the
+`/api/v1/sbom?requests={id1},{id2},..` endpoint conforms to the schema. The only part of the SBOM 
+that Cachito *does* pre-fill with relevant data is the `components` attribute.
```

Comment:
Most of this is only applicable to ICMs (describing containers, the whole paragraph about Cachito not knowing about containers).

IMO the points we want to document are

* the SBOM is CycloneDX
* it includes all the components (packages and dependencies) for the specified requests
* the purls follow the same rules as those in ICM

Reply:
ok